src/sendMessage.js

```diff
@@ -182,6 +184,26 @@ module.exports = function(defaultFuncs, api, ctx) {
     }
     cb();
   }
+  
+  function handleEmoji(msg, form, callback, cb) {
+    if (msg.emoji) {
+      if(msg.emojiSize == null)
+      {
+        return callback({error: "emojiSize property is empty"});
+      }
+      if(msg.emojiSize != "mini" && msg.emojiSize != "medium" && msg.emojiSize != "large")
+      {
+        return callback({error: "emojiSize property is invalid"});
+      }
+      if(msg.emoji.length > 3)
+      {
+        return callback({error: "Invalid emoji"});
+      }
+      form['message_batch[0][body]'] = msg.emoji;
```

Comment:
Won't this silently overwrite the body of a message?  I'd like to give an error or at least a warning if someone tries to do this.